format-specs/compatible-parquet.md

```diff
@@ -0,0 +1,42 @@
+
+# Parquet Geospatial Compatibility
+
+The goal of GeoParquet is that every tool producing Parquet and includes geospatial data uses our [defined metadata](./geoparquet_spec) to enable maximum compatibility. But in the interim time when only some tools properly produce GeoParquet, we are recommending the following best practices for the distribution of geospatial data as Parquet. 
+
+This is *only* recommended for those who are using tools that don't yet produce valid GeoParquet, and we encourage advocating to the creaters of tools you are using to implement the GeoParquet spec. Feel free to [start a discussion](https://github.com/opengeospatial/geoparquet/discussions) to raise awareness of tools that ideally support Geoparquet - the community can likely help in encouraging an implementation.
+
+Tools and libraries that read GeoParquet are encouraged to be able to parse these files, to make it easy to get data into the GeoParquet ecosystem. But it is only recommended for tools and libraries to produce valid GeoParquet, following [Postel's Law](https://en.wikipedia.org/wiki/Robustness_principle) of being liberal in what you accept but conservative in what you send.
+
+## Compatibility Guidelines
+
+TODO: flesh this out with more narrative.
+
+The geometry column should be named 'geometry'.
+
+The geometry column should be either a binary column with Well Known Binary (WKB) or a text column with Well Known Text (WKT) used to define the geometries.
```

Comment:
I believe this was raised in a past discussion, but it is a bit paradoxical to allow WKT geometries in a "compatible GeoParquet" file, whereas the GeoParquet itself only supports WKB.

Reply:
I think if we were calling this thing like 'simple geoparquet' or something with it in the name I'd agree we should put it in the spec. I see this as just a set of recommendations.

But I'd be up to call it WKB in this 'core' description, and then mention WKT as an option to support.

Reply:
Moved WKT to not be in the 'core' description, but an option that readers might add.

Reply:
This came up in the WKT vs WKB discussion as well, and I think if we want to allow WKT here in the "compatible" files, we should just officially add it as a possible encoding in the spec (https://github.com/opengeospatial/geoparquet/discussions/170#discussioncomment-5817223)

Reply:
So my compromise on this was to not include it in the main 'description' of a compatible file, but to just mention it as an option some readers could do. It says 'Readers can optionally support user input (in whatever manner works for the tool / library) to provide hints for metadata that is not inline with the lowest common denominator compatibility. This would include things like letting the user supply a geometry column name (ie something other than 'geometry'), using Well Known Text (WKT) in a `STRING` column instead of WKB'

I lean towards **not** including it in the core spec, since if it was there I'd want to strongly recommend people *don't* use it, since WKB is better in most ways. And it seems nicer to just give one clear target, not two that readers have to support. I see the compatible files as not 'GeoParquet', just a way to format Parquet. Like if we called it 'GeoParquet Simple' or something like that (which we decided not to for good reasons) then I could see it making a lot of sense to make sure the WKT option is in the core spec. But I'm viewing this as not so directly related, and to just suggest WKT as something readers could do to help people get data in - not even as the core 'compatibility' route.

Reply:
OK, I am fine with keeping it mentioned here without adding it "officially" to the spec. We certainly want to avoid that libraries claim to support GeoParquet when they only or by default write WKT.